package.json

```diff
@@ -133,20 +133,20 @@
     "@angular/platform-browser": "7.2.15",
     "@angular/platform-browser-dynamic": "7.2.15",
     "@angular/router": "7.2.15",
-    "@blackbaud/auth-client": "2.21.0",
```

Comment:
In this case (and possibly other cases) we should update the min version in `peerDependencies` to match the version in `devDependencies`. That way we don't assume a new feature during development and then allow the consumer to install an earlier version without the new feature.

Reply:
Good point. My other RC branch will modify the peer dependency versions of Angular/webpack, etc.